src/helpers/simplify_claims.ts

```diff
@@ -6,7 +6,7 @@ import type { SimplifiedClaim, SimplifiedClaims, SimplifiedPropertyClaims, Simpl
 
 function simplifySnaks (snaks, options) {
   const { propertyPrefix } = options
-  const simplifiedSnaks = {}
+  const simplifiedSnaks: any = {}
```

Comment:
```suggestion
  const simplifiedSnaks: Record<string, any> = {}
```

Reply:
The whole file is a typing mess… I just tried to get rid of `// @ts-nocheck` and `any` seemed a bit better than ignoring whole files…

The typings are not ideal currently but as there are tests currently we know the methods aren't doing horrible stuff so… Also this PQ is already big enough 😬